### ISSUE TITLE
where do I need to put the modification about X-sendfile ?

### ISSUE THREAD

**Issue by hugoheitz** [sbdomo/MyReadings#11]
on the FAQ website about DSM 5 modifications needed, you speak about X-sendfile but I dont know exactly where to put it in the httpd-conf-user file.

Can you be more specific ?
thanks

**Comment by sbdomo** [comment]
For example, at the end of the file.
Save the original before...

**Comment by hugoheitz** [comment]
I created a myreadings-dev version on my NAS. great to test evolutions.
I tried to modify the file for XSend-File because it would be lighter on the server side to handle the transfers.
I added /volume1 to the existing line (like this) :
 XSendFilePath /var/services/web /volume1
And I cant restart the webserver.
I see in your message 
http://www.nas-forum.com/forum/topic/40675-configuration-xsendfile-sous-dsm-5/
that you put :
 XSendFilePath /volume1/  
Can I add /volume1/ to the existing line, what is the correct solution ?

thanks for your help,
